contracts/test/ERC20.t.sol

```diff
@@ -10,240 +10,243 @@ import { MockERC20 } from "./mocks/MockERC20.sol";
 
 contract ERC20BaseTest is TestUtils {
 
-    bytes constant ARITHMETIC_ERROR = abi.encodeWithSignature("Panic(uint256)", 0x11);
+    address internal immutable self = address(this);
 
-    MockERC20 token;
+    bytes internal constant ARITHMETIC_ERROR = abi.encodeWithSignature("Panic(uint256)", 0x11);
 
-    address internal immutable self = address(this);
+    MockERC20 internal _token;
 
-    function setUp() public virtual {
-        token = new MockERC20("Token", "TKN", 18);
+    function setUp() external virtual {
```

Comment:
The style guide has the test functions marked as public, but not sure what the convention is.

Reply:
ight be just a copy and paste where it wasn't considered.